src/cli/command.rs

```diff
@@ -41,10 +61,22 @@ pub enum Command {
     #[arg(short, long)]
     schema: bool,
 
-    /// Path of the file to save final configuration in
-    #[arg(short, long("out"))]
-    out_file_path: Option<String>,
+    /// Display the generated blueprint in the console
+    #[arg(short, long)]
+    blueprint: bool,
   },
+
+  /// Merge multiple configuration file into one
+  Compose {
+    /// Path for the configuration files separated by spaces if more than one
+    #[arg(required = true)]
+    file_path: Vec<String>,
+
+    /// Format of the result. Accepted values: JSON|YML|GQL. For blueprint option, only YML and JSON are allowed
```

Comment:
```suggestion
    /// Format of the result. Accepted values: JSON|YML|GQL.
```
Compose doesn't support blueprint anymore, right?